tt_metal/impl/debug/dprint_server.cpp

```diff
@@ -265,30 +278,37 @@ void DebugPrintServerContext::WaitForPrintsFinished() {
 void DebugPrintServerContext::AttachDevice(Device* device) {
     chip_id_t device_id = device->id();
 
-    // A set of all valid worker cores, used for checking the user input. Note that the core coords
+    // A set of all valid prinatble cores, used for checking the user input. Note that the coords
     // here are physical, which matches the user input for configuring the print server.
-    set<CoreCoord> all_worker_cores;
+    set<CoreCoord> all_printable_cores;
+    // The set of all printable cores is Tensix + Eth cores
     CoreCoord logical_grid_size = device->logical_grid_size();
     for (uint32_t x = 0; x < logical_grid_size.x; x++) {
         for (uint32_t y = 0; y < logical_grid_size.y; y++) {
             CoreCoord logical_coord(x, y);
             CoreCoord worker_core = device->worker_core_from_logical_core(logical_coord);
-            all_worker_cores.insert(worker_core);
+            all_printable_cores.insert(worker_core);
         }
     }
+    set<CoreCoord> all_eth_cores;
+    for (const auto& eth_core : device->get_active_ethernet_cores()) {
```

Comment:
can you change this to `device->ethernet_cores()`.  Just prefetching the need to change this later when we have FW running on "inactive cores" as well

Reply:
Done in latest commit